Changelog.md

```diff
@@ -1,5 +1,57 @@
 # Changelog
 
+## 20231009
+
+#### New schemas:
+- LoyaltiesListMemberRewardsRequestQuery
+- LoyaltiesGetPointsExpirationRequestQuery
+- LoyaltiesGetPointsExpirationResponseBody
+- LoyaltiesListCardTransactionsRequestQuery
+- LoyaltiesListCardTransactionsResponseBody
+- LoyaltyCardTransactionsType
+- SimpleLoyaltyVoucher
+- LoyaltyCardTransaction
+- LoyaltyCardTransactionsFields
+- LoyaltiesExportCardTransactionsRequestBody
+- LoyaltiesExportCardTransactionsResponseBody
+- RewardAssignment
+- Reward
+- RewardTypeCoin
+- RewardTypeMaterial
+- RewardTypeCampaign
+- RewardType
+- LoyaltiesAddOrRemoveCardBalanceResponseBody
+- LoyaltiesTransferPointsResponseBody
+- LoyaltiesTransferPoints
+- LoyaltiesTransferPointsRequestBody
+- LoyaltiesListMemberRewardsResponseBody
+- LoyaltiesAddOrRemoveCardBalanceRequestBody
+- PointsExpirationTypes
+
+#### Schemas changes
+- /v1/loyalties/{campaignId}/members/{memberId}/balance
+    - new request schema `LoyaltiesAddOrRemoveCardBalanceRequestBody` (old `8_req_add_remove_points_balance`)
+    - new response schema `LoyaltiesAddOrRemoveCardBalanceResponseBody` (old `8_res_add_remove_points_balance`)
+- /v1/loyalties/members/{memberId}/balance
+    - new request schema `LoyaltiesAddOrRemoveCardBalanceRequestBody` (old `8_req_add_remove_points_balance`)
+    - new response schema `LoyaltiesAddOrRemoveCardBalanceResponseBody` (old `8_res_add_remove_points_balance`)
+- /v1/loyalties/{campaignId}/members/{memberId}/transfers
+    - new request schema `LoyaltiesTransferPointsRequestBody` (old `8_req_transfer_loyalty_points`)
+    - new response schema `LoyaltiesTransferPointsResponseBody` (old `8_obj_loyalty_card_object_non_expanded_categories`)
+- /v1/loyalties/{campaignId}/members/{memberId}/transactions
+    - new request schema `LoyaltiesListCardTransactionsRequestBody` (old `8_res_get_loyalty_card_transactions`)
+- /v1/loyalties/members/{memberId}/transactions
+    - new request schema `LoyaltiesListCardTransactionsResponseBody` (old `8_res_get_loyalty_card_transactions`)
+- /v1/loyalties/members/{memberId}/transactions/export
+    - new request schema `LoyaltiesExportCardTransactionsRequestBody` (old `8_req_create_loyalty_card_transactions_export`)
+    - new response schema `LoyaltiesExportCardTransactionsResponseBody` (old `8_obj_export_transactions_object`)
```

Comment:
I see some differences in `conditions` and `result` attributes... please, confirm if that is correct.

![image](https://github.com/voucherifyio/voucherify-openapi/assets/35666945/da4a9f65-bf18-4344-b78c-40a02098477e)

Reply:
that is correct, this filter is based on memberId from url param. we cannot add additional filters